src/mixxx.cpp

```diff
@@ -1283,6 +1289,20 @@ void MixxxMainWindow::initActions()
     connect(m_pViewShowPreviewDeck, SIGNAL(toggled(bool)),
             this, SLOT(slotViewShowPreviewDeck(bool)));
 
+    QString showEffectsTitle = tr("Show Effects Rack");
```

Comment:
Hm, "Show Effects Rack" or "Show Effect Rack"?  The latter feels more fluid to me.